docs/development/README.md

```diff
@@ -74,6 +75,29 @@ chmod +x .git/hooks/pre-commit
 _Please note that the Github workflow will still prevent this from merging
 unless the CHANGELOG is updated._
 
+### Pocket Network CLI
+
+Pocket provides a CLI for interacting with Pocket Core. The CLI is meant to be an user but also a machine friendly way for interacting with Pocket Network.
+
+The commands available are listed [here](../../app/client/cli/doc/commands/client.md).
```

Comment:
You should refer to `pocket/rpc/doc/README.md` (that's where the full documentation is)

Reply:
Updated reference